devtools-ci-index.yaml

```diff
@@ -1483,6 +1483,17 @@
             - *gpgkeys-for-signing-maven-artefacts
     <<: *job_template_defaults
 
+- job-template:
```

Comment:
We had to create a new template because the default template doesn't have `fabric8-hub-token` which we need to add success/failure comment on Github PR.